RCPTT_Tests/Contexts/ExperimentalDetailsProcedures.ctx

```diff
@@ -0,0 +1,30 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Context-Type: org.eclipse.rcptt.ctx.ecl
+Element-Name: ExperimentalDetailsProcedures
+Element-Type: context
+Element-Version: 2.0
+Id: _hamJ8FDbEee8yMBD6nu81w
+Runtime-Version: 2.1.0.201606221726
+Save-Time: 6/14/17 2:02 PM
+
+------=_.ecl.context-718f04b4-ed39-33e3-af62-0995e4561998
+Content-Type: text/ecl
+Entry-Name: .ecl.context
+
+// Procedures to interact with experimental details
+
+// set the rb number
+proc "set_rb_number" [val new_rb_number] [val use_enter -value false] {
+    with [get-view "Experiment Details"] {
+        get-editbox -after [get-label "RB Number"] | set-text $new_rb_number
+        if $use_enter -then {
+            get-button Set | check_button_enabled_and_click
```

Comment:
Doesn't work for me. Needs to be `check_button_enabled_and_click [get-button Set]`